starter-code/functions-and-arrays.js

```diff
@@ -84,7 +122,12 @@ console.log(uniqued);
 
 // Finding Elements
 function doesWordExist (wordsArray, word) {
-
+  var biggest="";
+  for ( var ix=0; ix < words.length; ix++){
+    if ( biggest.length < words[ix].length){
```

Comment:
You don't want to find the biggest word, you want to check if that word is in the array.